index.mjs

```diff
@@ -124,7 +124,7 @@ export async function question(query, options) {
   return answer
 }
 
-export async function fetch(url, init) {
+export function fetch(url, init) {
```

Comment:
Purpose of async keyword here is readability. It makes clear for the reader what this func is async.

Reply:
I also think that nodeFetch is async: https://www.npmjs.com/package/node-fetch (the library used)